src/frontend/apps/lti_site/components/App/AppInitializer/index.tsx

```diff
@@ -55,11 +55,12 @@ export const AppInitializer = (
   const isFeatureEnabled = useIsFeatureEnabled();
 
   const decodedJwt = useMemo(() => {
-    if (jwt && isJwtInitialized) {
```

Comment:
Reading the commit description I don't understand why you made this modification. What is the motivation.